### ISSUE TITLE
improve color contrast in the favicon → switch from yellow on white to white on black

### ISSUE THREAD

**Issue by gruns** [lunal-dev/home#41]
current favicon is yellow on white. bright on bright. contrast is poor. https://confidential.ai/favicon.ico

in turn, looks bad on google:

<img width="1476" height="422" alt="Image" src="https://github.com/user-attachments/assets/292fc84c-01e7-4d31-b78d-642ca72fb6f0" />

todo: update it to be white on black. like our github logo: 

<img width="48" height="48" alt="Image" src="https://github.com/user-attachments/assets/c54a452d-1198-449e-9678-244a8adaca4f" />
